doc/contributing.md

```diff
@@ -28,3 +38,106 @@ If possible, provide:
    and Python version.
 2. A minimal way to reproduce the problem you see; these can be either
    a set of instructions, or a script.
+
+## Process guidelines
+
+Because many developers work on SkyPortal, and PRs sometimes come in
+at a rapid pace, we have guidelines to streamline review and
+development:
+
+### Code style
+
+We don't like arguing about code style, and likely you don't
+either. Therefore, we use code formatters: black for Python, and
+Prettier for JavaScript.  Code is an art, and opinions differ of what
+looks good: we choose to spend our time writing correct, elegant code.
+
+### Testing
+
+All functionality should be accompanied by tests.  We use pytest and
+Selenium with Geckodriver.  PRs can only be merged once tests have
+been added and pass.  The continuous integration system indicates this
+with a green checkmark, hence you may see developers talking about "PR
+599 being green" ✅.
+
+### Reviews
+
+All code that goes into SkyPortal is reviewed by two team members
+(team members are persons with commit access) or, if the contributor
+is a team member, by at least one other team member.  We find review
+invaluable for improving code quality, and no author ever merges their
+own work.
+
+We ask all reviewers to be mindful that there is a human at the other
+side of the PR.  As such, consider the review process as a
+conversation aimed at getting the work mergeable as quickly as
+possible:
+
+- Do not nitpick in comments; add small fixes as suggestions (click
+  the `±` button) or push uncontroversial changes as a commit into
```

Comment:
This just adds a suggested change, but doesn't actually commit it yet

Reply:
@acrellin I wasn't sure what you wanted to see here, but I added some language---take a look and let me know.